src/multibody/model.hxx

```diff
@@ -72,12 +72,12 @@ namespace se3
     nq += joint_model.nq();
     nv += joint_model.nv();
 
-    if (D::NV == -1)
+    if (D::NV == Eigen::Dynamic)
```

Comment:
If this is necessary here, then it also is in CRBA. You should have consistent design.
I think it is never necessary.